packages/lit-analyzer/src/analyze/util/directive/is-lit-directive.ts

```diff
@@ -1,5 +1,7 @@
 import { SimpleType, SimpleTypeKind } from "ts-simple-type";
 
+const partTypeNames: ReadonlySet<string | undefined> = new Set(["Part", "NodePart", "AttributePart", "BooleanAttributePart", "PropertyPart"]);
```

Comment:
Either add `EventPart` to this list, or instead check if the argument type is assignable to `Part`.